maven-java-11/pom.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Need to add this module to the root pom.xml, and then in this pom.xml, need to aggregate the multimodule-maven-project.

Reply:
Hey @KevinGilmore No worries. I'll do that today and post accordingly.